frontend/src/routes/Video.tsx

```diff
@@ -343,26 +354,27 @@ const Metadata: React.FC<MetadataProps> = ({ id, event }) => {
                 <ShareButton event={event} />
             </div>
         </div>
-        <hr />
         <div css={{
             display: "grid",
             gridTemplate: "1fr / 1fr fit-content(30%)",
-            columnGap: 48,
-            rowGap: 24,
+            gap: 16,
```

Comment:
I think right now I would probably use a flexbox with `flex-wrap`. The current sizing logic is too restrictive and quickly looks bad for certain videos. I would try to avoid any fixed widths or stuff like that. In other words: the exact widths of the two childs can vary from video to video, depending on the content. When there is a big description, that box should be larger; if there is no description, it's fine if it's getting fairly small. I think it should be possible to do all that with the `flex-grow` and `flex-shrink` options (of course, likely using the `flex:` shorthand). And in particular: also trying out using values other than 1 and 0, such that a box might shrink or grow faster than the other.
This is certainly a bit of experimentation.

The "Tour of the moon" video on the test deployment has a proper long description. Try testing with that as well.
For "Chicken", the current design overflows the page, on the smallest screen width where the nav bar is still visible.

Reply:
This already improved a lot. Some comments from just testing the deployment:

- For "Tour of the Moon" I find it absolutely fine for all screen widths
- For videos without descriptions I find it a bit weird that the metadata box gets so wide. Well, probably only because the content is narrower than the container.
- Maybe rather set `flex-basis` of the table to `max-content`, set `flex-grow` to 1 (and the `flex-grow` of the description div to something at least like 3 times higher), and `flex-shrink` to 1?

But I will have a look at the code still, from the dev tools it seemed like there was some interesting stuff going on.

Reply:
> For videos without descriptions I find it a bit weird that the metadata box gets so wide.

Hm yeah I made that in reaction to your previous comment (`When there is a big description, that box should be larger; if there is no description, it's fine if it's getting fairly small.`), but might have overdone it...

> Maybe rather set flex-basis of the table to max-content, set flex-grow to 1 (and the flex-grow of the description div to something at least like 3 times higher), and flex-shrink to 1?

So right now the `flex-grow` of the description box is dependent on whether there is a description (`flex-grow`: twice that of the metadata box) or not (half of metadata's `flex-grow`). One could of course argue that this doesn't make too much sense since there might also be a very short description that's not that much more text than the `no description` placeholder. So yeah I might need to find a different solution for that or use your above suggestion.
`flex-shrink` of both is always `1` right now, but doesn't ever take effect since `flex-wrap: wrap` will always wrap the boxes before they can shrink. The different `flex-bases` basically act as a break point for the `wrap` to take effect. 
Maybe this will make more sense when you take a look at the code (or maybe not^^).

> from the dev tools it seemed like there was some interesting stuff going on.

That's one way to put it for sure :D